clash-prelude/tests/Clash/Tests/Signal.hs

```diff
@@ -240,8 +247,8 @@ case_dynamicHasEffect = do
 
     clk11 = clockGen @H11
     clk77 = clockGen @H77
-    dclk11 = dynamicClockGen @H11 (pure (hzToPeriod 11))
-    dclk77lying = dynamicClockGen @H77 (pure (hzToPeriod 78))
+    dclk11 = dynamicClockGen @H11 (pure (hzToFs 11))
```

Comment:
Why can we use `hzToFs` here, and not do the roundabout way of doing `1000 * hzToPeriod 11` like we did earlier? Perhaps worth a comment why this is different.

Reply:
Good catch, we can't!